vendor/bat-native-ledger/src/bat/ledger/internal/media/helper.cc

```diff
@@ -68,4 +69,40 @@ std::string ExtractData(const std::string& data,
   return match;
 }
 
+void GetVimeoParts(
+    const std::string& query,
+    std::vector<std::map<std::string, std::string>>* parts) {
+  base::Optional<base::Value> data = base::JSONReader::Read(query);
+  if (!data || !data->is_list()) {
+    return;
+  }
+
+  for (auto& item : data->GetList()) {
```

Comment:
Good to make `item` `const` here, if it's possible.

Reply:
done